core/src/java/org/hypergraphdb/HGEnvironment.java

```diff
@@ -1,21 +1,23 @@
-/* 
```

Comment:
why are there so many cosmetic changes? editor is saving extra newline characters or something?